CONTRIBUTING.md

```diff
@@ -67,13 +67,13 @@ as part of the verification process, that is applied to all new contributions.
 ## Issues and Pull Requests
 
 Issues within MiNiFi C++ are tracked in the official [Apache JIRA](https://issues.apache.org/jira/projects/MINIFICPP/issues).
-New users can register freely and ask for contributor access on the
-[Developers Mailing List](https://nifi.apache.org/mailing_lists.html).
-Unassigned tickets may be assigned to yourself or filed via this JIRA instance. Pull requests can be submitted via our
-[Github Mirror](https://github.com/apache/nifi-minifi-cpp) . When doing so please create a JIRA issue for your pull
-request.
+New users can freely request JIRA access on the
+[Self-Service Portal](https://selfserve.apache.org/jira-account.html) for the nifi project. An Apache PMC member will review and
+approve the request as prudent. Unassigned tickets may be assigned to yourself or filed via this JIRA instance. As a new user looking to contribute,
+any tickets that are tagged as "beginner", "starter", or "newbie" are good places to start. Pull requests can be submitted via our
+[Github Mirror](https://github.com/apache/nifi-minifi-cpp). If your contribution does not have a corresponding JIRA issue for it then please create one.
```

Comment:
it's not really a mirror; maybe just `... can be submitted via [GitHub](...)`?

Reply:
It's a mirror of https://gitbox.apache.org/repos/asf/nifi-minifi-cpp.git

But there is two-way sync, and we're not really using the ASF GitBox interface, so this detail can be omitted